spec/lib/saml_idp/metadata_builder_spec.rb

```diff
@@ -30,12 +30,5 @@ module SamlIdp
       )
       expect(Saml::XML::Document.parse(subject.signed).valid_signature?(cloudhsm_idp_x509_cert_fingerprint)).to be_truthy
     end
-
-    it "includes logout element" do
```

Comment:
should we convert this to a negative test? ex `it does not include a logout element`?